builders/build/eth-api/dev-env/brownie.md

```diff
@@ -204,14 +204,17 @@ If you want to specify the compiler version or compilation settings, you can do
 
 ```yaml
 compiler:
-  evm_version: london
+  evm_version: null
```

Comment:
Why setting this to null?

Reply:
cause this will default to the latest evm_version that brownie supports, which as of right now is istanbul. 

but i figured i would leave it there so it shows the user they can update the evm version there and then the !!! note below points to the supported versions, in case brownie adds support for some of the newer compiler versions at some point